docs/adding_new_models.md

```diff
@@ -0,0 +1,12 @@
+# Adding new models
+
+Currently, you must clone the HELM repostory and modify the HELM code locally to add a new model.
+
+1. Pick a new organization name that does not conflict with any of the organizations for the [existing models](models.md).
+2. Define a new [`Model`](https://github.com/stanford-crfm/helm/blob/v0.2.0/src/helm/proxy/models.py#L39) describing your model and add it to [`ALL_MODELS` in `helm.proxy.models`](https://github.com/stanford-crfm/helm/blob/main/src/helm/proxy/models.py#L88). Set `Model.group` and `Model.creator_organization` to your organization name. Set `Model.name` to  `"your_organization_name/your_model_name"`.
```

Comment:
Does this have to be pinned to `v0.2.0`?  Seems like we'd have to update this every time?

Reply:
We need to pin this to some githash anyway (because they're specific lines). We only need to change this if the API or instructions change significantly; minor changes are not a big deal.